src/browser/web-view.js

```diff
@@ -327,7 +329,7 @@ define((require, exports, module) => {
       display: 'none'
     },
     passive: {
-      visibility: 'collapsed'
+      visibility: 'collapse'
```

Comment:
Hmm, this might be relying on a bug in our implementation. From https://developer.mozilla.org/en-US/docs/Web/CSS/visibility.

> For table rows, columns, column groups, and row groups the row(s) or column(s) are hidden and the space they would have occupied is removed (as if display: none were applied to the column/row of the table). __For other elements, collapse is treated the same as hidden.__